slider/library.py

```diff
@@ -259,13 +261,19 @@ def save(self, data, *, beatmap=None):
         if beatmap is None:
             beatmap = Beatmap.parse(data.decode('utf-8-sig'))
 
-        path = self.path / (
+        beatmap_file_name = (
             f'{beatmap.artist} - '
             f'{beatmap.title} '
             f'({beatmap.creator})'
             f'[{beatmap.version}]'
             f'.osu'
         ).replace('/', '_')
+        if _platform.startswith("win"): beatmap_file_name = _sub(
+            '[:*?"<>|]', 
```

Comment:
this is interpreted as a regex and everything but `:<>` are special in regex. Maybe we should just use a little loop and `str.replace`